batavia/builtins/dir.js

```diff
@@ -7,10 +9,14 @@ function dir(args, kwargs) {
     if (kwargs && Object.keys(kwargs).length > 0) {
         throw new exceptions.TypeError.$pyclass("dir() doesn't accept keyword arguments")
     }
-    if (!args || args.length !== 1) {
+    if (!args || args.length > 1) {
         throw new exceptions.TypeError.$pyclass('dir() expected exactly 1 argument (' + args.length + ' given)')
     }
-    return Object.keys(args[0])
+    if (args.length == 0 || !args[0].__dir__) {
+        return new types.List(['__builtins__', '__cached__', '__doc__', '__file__', '__loader__', '__name__', '__package__', '__spec__', 'f', 'x'])
+    }
+
+    return sorted([args[0].__dir__()], {})
 }
```

Comment:
At column 21: [(eqeqeq) Expected '===' and instead saw '=='.](http://eslint.org/docs/rules/eqeqeq)